apps/wing-console/console/ui/src/ui/endpoint-tree.tsx

```diff
@@ -0,0 +1,77 @@
+import { LinkIcon } from "@heroicons/react/24/outline";
+import {
+  ScrollableArea,
+  Toolbar,
+  TreeItem,
+  TreeView,
+  useTheme,
+} from "@wingconsole/design-system";
+import classNames from "classnames";
+
+import { EndpointItem } from "../shared/endpoint-item.js";
+
+import { NoEndpoints } from "./no-endpoints.js";
+
+export interface EndpointTreeProps {
+  endpointList: EndpointItem[];
+}
+
+export const EndpointTree = ({ endpointList }: EndpointTreeProps) => {
+  const { theme } = useTheme();
+
+  return (
+    <div
```

Comment:
if moved to the leftPanel section, we won't have that much space. I think making the endpoint label linkable would be better. we can have the url value as a tooltip (I don't see a reason to have it displayed in the panel directly)